### DISCUSSION TITLE
DAG's Next Run date not updating

### DISCUSSION THREAD

**Discussion by vandanthaker** [apache/airflow#25501] [Q&A] [answered]
Airflow version - 2.2.2

I am seeing an issue for my couple of dags, where Next Run date is not updating even though my scheduler is healthy and running fine. Due to this schedules are getting missed. I'm not seeing any particular errors as well in the logs. Can someone please guide me on how to debug the issue here?

![image](https://user-images.githubusercontent.com/11852117/182538539-ccf976a6-c5a0-4382-8c95-83655144dffa.png)

![image](https://user-images.githubusercontent.com/11852117/182538730-33d4c50d-5362-4f41-9344-1f8115478892.png)

**Comment by Taragolis** [comment, marked as answer]
@vandanthaker I would started with this:

1. Did you pause this DAG?
2. What actually you've specified in  `start_date` and `end_date` in your DAG during creation?

**Reply by vandanthaker** [reply]
No DAG was never paused. Checked audit logs to confirm

start_date=days_ago(0). haven't specified end_date explicitly

**Reply by Taragolis** [reply]
`start_date=days_ago(0)` this is the root of [your issue](https://airflow.apache.org/docs/apache-airflow/stable/faq.html?highlight=start_date#why-is-task-not-getting-scheduled)


I would recommend use static value for `start_date` as well as `end_date` if required.

In your case you basically get current timestamp by call [days_ago(0)](https://github.com/apache/airflow/blob/cd14f3f65ad5011058ab53f2119198d6c082e82c/airflow/utils/dates.py#L251-L264) and you've never achieve:
* Is your start_date set properly? The Airflow scheduler triggers the task soon after the start_date + schedule_interval is passed.

**Comment by unihon** [comment]
<img width="1600" alt="image" src="https://user-images.githubusercontent.com/42078581/198801013-f13d7baa-a01d-41f8-8e66-e107a9f60f47.png">


<img width="1601" alt="image" src="https://user-images.githubusercontent.com/42078581/198794755-0c769b99-5521-4e42-a556-75dd7831acf8.png">


My problem is somewhat similar to yours, the task cycle works fine, but occasionally the scheduling of part of the cycle is missing. As shown in the picture, the dagrun of `2022-10-28, 23:00:00` is missing! I don't know why, can anyone explain this


info:
start_date: days_ago(0)
catchup: False
schedule_interval: '0 * * * *'
